assets/js/romo/datepicker.js

```diff
@@ -67,7 +68,13 @@ RomoDatepicker.prototype.doBindElem = function() {
       this._hide(this.indicatorElem);
     }
     this.indicatorElem.on('click', $.proxy(this.onIndicatorClick, this));
-    this.elem.css({'padding-right': '22px'});
+
+    var indicatorWidthPx   = this.elem.data('romo-datepicker-indicator-width-px') || this.defaultIndicatorWidthPx;
+    // left-side spacing
+    // + caret width
```

Comment:
@kellyredding - minor, indicator instead of caret to line up with the variable names